main_test.go

```diff
@@ -72,7 +72,7 @@ func Test_run(t *testing.T) {
 	}
 }
 
-var re = regexp.MustCompile(`Code generated by .*deep-copy.*; DO NOT EDIT.`)
```

Comment:
If this diff is introduced without the corresponding [proposed solution](https://github.com/globusdigital/deep-copy/pull/43/files#diff-6ae5969c5a30f999a4a26bd3e4121c43a727e4afd41689671e35492197e02560R173) the tests fail for the same reason our CI is failing